package.json

```diff
@@ -15,25 +15,28 @@
         "url": "git+https://github.com/twinssbc/Ionic2-Calendar.git"
     },
     "scripts": {
-        "build": "rm -rf dist && node_modules/.bin/ngc -p tsconfig.json",
-        "copy_static_files": "cp -r package.json README.md LICENSE tsconfig.json dist/",
+        "build": "rimraf dist && npx ngc -p tsconfig.json && npm run copy_static_files",
+        "copy_static_files": "cpx package.json dist/ &&  cpx README.md dist/ && cpx LICENSE dist/ && cpx tsconfig.json dist/",
```

Comment:
Why does this introduce two new dependencies for basic shell commands? Windows support?

Reply:
Yes, correct. I am working with Windows and to support every Distro we should use node Deps to fix this. 
So now it is working with Mac, Linux and Windows.